src/components/dropdowns/dropdowns.html

```diff
@@ -0,0 +1,61 @@
+<section class="docs-block">
+    <aside class="docs-block__info">
+        <h3 class="docs-block__header">Dropdowns</h3>
+    </aside>
+    <div class="docs-block__content">
+        <div class="mint-dropdown">
+            <div class="mint-dropdown__label">All options</div>
```

Comment:
@JanuszSzewczyk agreed, we should update label's font-family, then i could create new modifier:)
i created an issue: https://github.com/brainly/style-guide/issues/266